ror-tools/src/test/scala/tech/beshu/ror/tools/utils/TestEsContainerManager.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools.utils
+
+import cats.data.NonEmptyList
+import monix.eval.Task
+import monix.execution.atomic.AtomicInt
+import tech.beshu.ror.utils.containers.*
+import tech.beshu.ror.utils.containers.EsContainerCreator.EsNodeSettings
+import tech.beshu.ror.utils.containers.images.ReadonlyRestWithEnabledXpackSecurityPlugin
+import tech.beshu.ror.utils.containers.images.domain.Enabled
+
+import java.io.Console as _
+import scala.concurrent.duration.*
+import scala.language.postfixOps
+
+class TestEsContainerManager extends EsContainerCreator {
+
+  private val uniqueClusterId: AtomicInt = AtomicInt(1)
+
+  lazy val esContainer: EsContainer = createEsContainer
+
+  def start(): Task[Unit] = Task.delay(esContainer.start())
```

Comment:
I wonder if we have to manage the lifecycle of the container on our own. I understand that we don't need the container when we copy the tar from it, but does it hurt us that it's working while the tests are running? Maybe we can leave the lifecycle management for the test containers and scala-test? WDYT?

Reply:
Refactored, this class now exposes only single public method
```
def withTestEsContainer(withStartedEs: EsContainer => Unit): Unit
```